tools/journal_proto_processor.hpp

```diff
@@ -287,6 +293,10 @@ class JournalProtoProcessor final {
   // messages.
   std::map<Descriptor const*, std::vector<std::string>>
       cs_interface_parameters_;
+  std::map<Descriptor const*, std::vector<std::string>>
```

Comment:
Comment these maps.  You don't get to piggyback on the comment for the preceding map.